From e48f18eba0eae199ba7bc8a6a09ebf39799447c1 Mon Sep 17 00:00:00 2001
From: Ted Meyer <tmathmeyer@chromium.org>
Date: Wed, 2 Jun 2021 05:35:22 +0000
Subject: [PATCH] Extend enable-accelerated-video-decode flag for linux to m93

It appears that there is insistence that the flag be totally removed or
have its end-milestone increased, I can't just let it wait until the
feature is working again and then re-enable it. So i've moved it to 93.

R=dalecurtis

Fixed: 1207478
Change-Id: I26a5e790cd390825516b4a4b6af88e89b2d4f4eb
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2918478
Reviewed-by: Dale Curtis <dalecurtis@chromium.org>
Reviewed-by: Ted Meyer <tmathmeyer@chromium.org>
Commit-Queue: Ted Meyer <tmathmeyer@chromium.org>
Cr-Commit-Position: refs/heads/master@{#888312}
---
 chrome/browser/flag-metadata.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/browser/flag-metadata.json b/chrome/browser/flag-metadata.json
index 67e0330436..34542671f2 100644
--- a/chrome/browser/flag-metadata.json
+++ b/chrome/browser/flag-metadata.json
@@ -1193,7 +1193,7 @@
   {
     "name": "enable-accelerated-video-decode",
     "owners": [ "media-dev@chromium.org" ],
-    "expiry_milestone": 90
+    "expiry_milestone": 93
   },
   {
     "name": "enable-accessibility-live-caption",
